Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN poetry install --no-dev
 WORKDIR /app
 COPY . /app
 
-CMD [ "poetry", "run", "gunicorn", "todo_app.app:create_app()", "-w", "2", "--threads", "2", "-b", "0.0.0.0:80" ]
+CMD poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:$PORT
```

Comment:
Minor but although this matches Heroku's requirements, it now breaks our recommended Readme commands since it doesn't specify a default port! One nice addition could be to add a line to set the default port value back to 80 here;
`ENV PORT=80` should achieve that but will still allow Heroku to override the value when they choose to